Models/Http/HeadersBuilder.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace Infrastructure\Models\Http;
+
+
+class HeadersBuilder
```

Comment:
I don't see what object it builds.

It should be just Header object.

Reply:
fixed